src/projectmanager_gui/LoginWindow.java

```diff
@@ -22,6 +22,10 @@
  * @author Louis W.
  */
 public class LoginWindow extends javax.swing.JFrame {
+    
+    // Attributes
+    public LogWindow logwind = new LogWindow();
+    private ProjectManager analyster = new ProjectManager();
```

Comment:
this line says analyster, I think I changed this when we were debugging though, so those changes should get implemented when we merge.